ajax/libs/rita/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "rita",
+  "version": "1.1.40",
+  "author": "Daniel C. Howe <daniel@rednoise.org>",
+  "description": "RiTa: A toolkit for natural language and computational literature",
+  "homepage": "https://rednoise.org/rita",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dhowe/RiTaJS.git"
+  },
+  "keywords": [
+    "nlp",
+    "natural language",
+    "language",
+    "writing",
+    "generative"
+  ],
+  "license": "GPL-3.0",
+  "npmName": "rita",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "/rita.min.js"
```

Comment:
@pvnr0082t you should at least modify our tests and the already exist libs to reflect this requirement.